topics/genome-annotation/tutorials/hpc-for-lsgc/tutorial.md

```diff
@@ -0,0 +1,400 @@
+> >
+> {: .solution }
+>
+{: .question}
+
+As we discussed in the previous section, running optimal aligning tools on such "small" data can be difficult (in fact, tools such as the well-known `EMBOSS needle` will require cuadratic amounts of time and memory, whereas tools such as `EMBOSS strecher` will require quadratic time ({% cite myers1988optimal %})). These limitations can make it impractical in many situations. Therefore, we will now learn how to overcome these limitations by employing seed-based methods, particularly `GECKO`.
+
+## Running the comparison
+
+We will now run a comparison between `mycoplasma hyopneumoniae 232` and `mycoplasma hyopneumoniae 7422` in Galaxy using `GECKO`.
+
+> ### {% icon hands_on %} Hands-on: Comparing two mycoplasmas with GECKO
+> 1. {% tool [GECKO](toolshed.g2.bx.psu.edu/repos/iuc/gecko/gecko/1.2) %} with the following parameters
+>    - {% icon param-file %} *"Query sequence"*: `232.fasta`
+>    - {% icon param-file %} *"Reference sequence"*: `7422.fasta`
+>    - *"K-mer seed size"*: `16`
+>    - *"Minimum length"*: `50`
+>    - *"Minimum similarity"*: `60`
+>    - *"Generate alignments file?"*: `Extract alignments (CSV and alignments file)`
+> 2. Check out the files that have been generated, i.e. the `CSV` and the `Alignments` file. 
+>    > ### {% icon question %} Questions
+>    >
+>    > 1. What information is provided in the `CSV` file?
+>    > 2. And in the `Alignments` file?
+>    > 3. What happens if we re-run the experiment with other parameters (e.g. change `Minimum length` to `5000` and `Minimum similarity` to `95`)?
+>    >
+>    > > ### {% icon solution %} Solution
+>    > > 1. The `CSV` file contains a summary of the detected High-Scoring Segment Pairs (HSPs) or alignments. The file is divided in a few rows of metadata (e.g. containing the sequence names) and one row per alignment detected. Check out the `GECKO` help (bottom of the tool {% icon tool %} page to know what each column does!
+>    > > 2. The `Alignments` file contains the actual regions of the query and reference sequence for each alignment detected. With this file, you can investigate individual alignments, find mutations and differences between the sequences, extract the aligned part, etc.
+>    > > 3. Changing the parameters affect the number of alignments GECKO will detect. In fact, if we use parameters that are too restrictive, then we might not get any alignments at all! On the other hand, if we use parameters that are too permissive, then we might get a lot of noise in the output. Thus, it is very important to understand what the parameters do. Never leave your parameters default, always know what they do! Check out the help section to get information about the parameters.
+>    >  {: .solution }
+>    {: .question}
+>    >
+>    > ### {% icon comment %} Note about parameters
```

Comment:
This comment block doesn't render fine.

Reply:
it'll be due to the second `>` on the above line.

Reply:
Fixed, there was an extra `>` before the `{: .comment}`
thanks again